packages/docs/content/getting-started/for-developers.mdx

```diff
@@ -0,0 +1,385 @@
+- Typescript definition files
+- Fonts and images
+
+The `dist` folder contains all the necessary files in the design system, your project should not be importing anything from `src`.
+
+```
+└── dist
+    ├── components/
+    │   └── index.js        Compiled JS entry point (CommmonJS)
+    ├── css/
+    │   └── index.css       Compiled CSS entry point
+    ├── esnext/
+    │   └── index.esm.js    Compiled JS entry point (ES Module)
+    ├── fonts/
+    ├── images/
+    ├── scss/
+    │   ├── base/           Base styles, HTML element selectors
+    │   ├── components/     Component styles
+    │   ├── settings/       Variables, mixins, and functions
+    │   ├── utilities/      Utility classes for individual CSS properties
+    │   └── index.scss      Precompiled SCSS entry point
+    └── types/              Typescript definition files
+```
+
+### Option 2: Reference assets from the CDN
+
+A content delivery network (CDN) is a collection of servers that dynamically allocate resources dependent on the requester's geographic location in order to optimize the distribution of assets.
+
+Some benefits and applications for utilizing the design system via CDN are:
+
+- Quickly utilize design system code and style assets for development/prototyping
+- The CDN caches assets for fast loading and scales automatically in case of a large number of requests
+- Ability to utilize a specific version of the design system easily
+
+#### Packages
+
+Packages which are currently available for reference:
+
+- **design-system** - The core CMSDS (versions 3.4.0 and above)
+- **ds-healthcare-gov** - Healthcare.gov themed DS (versions 7.4.0 and above)
+- **ds-medicare-gov** - Medicare.gov themed DS (versions 5.4.0 and above)
+
+#### URL structure
+
+```
+https://design.cms.gov/cdn/<package-name>/<version>/<resource>
+```
+
+Where `package-name` is one of the packages named above, `version` is the version number and `resource` is the file being requested from the resources listed above.
+
+#### Resources
+
+```
+  ├── css/
+  │   ├── index.css
+  │   ├── base/index.css
+  │   ├── components/index.css
+  │   └── utilities/index.css
+  └── js/
+      ├── react.production.min.js
+      ├── react-dom.production.min.js
+      └── bundle.js
+```
+
+#### Example for including version 3.4.0 Core CSS via CDN
+
+```html
+<link rel="stylesheet" href="https://design.cms.gov/cdn/design-system/3.4.0/css/index.css" crossorigin="anonymous" />
+```
+
+#### Example for including version 3.4.0 Core JS via CDN
+
+If you do not already have React and React-DOM included in your page, include them before the design system js bundle.
+
+```html
+<script type="text/javascript" src="https://design.cms.gov/cdn/design-system/3.4.0/js/react.production.min.js"></script>
+<script type="text/javascript" src="https://design.cms.gov/cdn/design-system/3.4.0/js/react-dom.production.min.js"></script>
+<script type="text/javascript" src="https://design.cms.gov/cdn/design-system/3.4.0/js/bundle.js"></script>
```

Comment:
@forrestbaer, I was just reading about `crossorigin="anonymous"`, and it seems like it would be good to recommend its use here on the `<script>` tags too. What do you think?

Reply:
If there's a performance cost, maybe we should take it off of the other example as well then. I figured it meant we could avoid sending all the cookies from healthcare.gov cross origin to design.cms.gov and that it would be a performance boost, but that was just a hunch. I don't have any experience with it.